### PR TITLE
example json improvements

### PR DESCRIPTION
**Pull request by jdan**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 4
```
Comment (justjake) [inline .prettierrc.json:2]: ???

File: pages/data.ts [hunk 2]
```diff
@@ -1,54 +1,6 @@
...
-    last_edited_by: "7c74a801-7cb5-4914-bbbe-2b18cd1ced76",
-    last_edited_time: 1563929940000,
-    parent_id: "c91b8f67-c73b-41c4-a0a8-95a9faa0a98d",
-    parent_table: "collection",
-    alive: true,
-}
+    title: "Hello, world!",
+    version: 4.0,
+    public: true,
+};
```
Comment (justjake) [inline pages/data.ts:6]: maybe you should have v1, v2, v3, v4, constants in here? or is that too much?

Probably too much.

File: .prettierrc.json (hunk 1 above)
Comment (jdan) [reply .prettierrc.json:2]: I guess this is wholly unnecessary huh, it doesn't magically enable prettier or anything

File: pages/data.ts (hunk 2 above)
Comment (jdan) [reply pages/data.ts]: might gave it away idk

**Review by justjake — APPROVED** [review]
I think keeping global.css as an empty file that's linked in would be fine, but also it's fine to remove.

**Pull request merged**